src/os/bluestore/BlueStore.cc

```diff
@@ -142,6 +142,10 @@ const string PREFIX_ZONED_CL_INFO = "G";  // (per-zone cleaner metadata)
 
 const string BLUESTORE_GLOBAL_STATFS_KEY = "bluestore_statfs";
 
+// Label offsets where they might be replicated. It is possible on previous versions where these offsets
+// were already used so labels won't exist there.
+const vector<uint64_t> bdev_label_offsets = {0, 1073741824, 107374182400, 1099511627776};
```

Comment:
what's about making this configurable through regular ceph settings?

Reply:
@aclamk What do you think about this?

Reply:
Caph settings can change, but to locate bdev label copies it is best to have them in fixed, predictable location.
I do not think that having this configurable brings additional value.